webapp/client/web_client.jsx

```diff
@@ -14,6 +17,7 @@ class WebClientClass extends Client {
     constructor() {
         super();
         this.enableLogErrorsToConsole(true);
+        this.connected = true;
```

Comment:
It seems kind of weird but "connected" really just means "has internet connection" and if you download this file then you should be "connected" already, so it starts as that. Also note that this isn't the WebSocket, just the regular web client

Think I should change the variable name and/or add a comment explaining?